code/computational_geometry/quickhull/Quick_Hull.c

```diff
@@ -0,0 +1,129 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+#include "bool.h"
+#include "geometry.h"
+#include <math.h>
+
+point first_point;		/* first hull point */
+
+convex_hull(point in[], int n, polygon *hull)
+{
+	int i;			/* input counter */
+	int top;		/* current hull size */
+	bool smaller_angle();
+	
+	if (n <= 3) { 		/* all points on hull! */
+		for (i=0; i<n; i++)
+                        copy_point(in[i],hull->p[i]);
```

Comment:
What is the implementation of this function?

Reply:
It is used in the main !